EmpleoDotNet/Controllers/TagController.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.Entity;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using EmpleoDotNet.Models;
+using EmpleoDotNet.Models.Repositories;
+using EmpleoDotNet.ViewModel;
+
+namespace EmpleoDotNet.Controllers
+{
+    public class TagController : EmpleoDotNetController
+    {
+        private readonly TagRepository _tagRepository;
+
+        public TagController()
+        {
+            _tagRepository = new TagRepository(_database); 
+        }
+
+        // GET: Tag
+        public ActionResult Index()
+        {
+            return View(_tagRepository.GetAllTags());
+        }
+
+        // GET: Tag/Details/5
+        public ActionResult Details(int? id)
+        {
+            if (id == null)
+            {
+                return new HttpStatusCodeResult(HttpStatusCode.BadRequest);
+            }
+            Tag tag = _tagRepository.GetTagById(id.Value);
+            if (tag == null)
+            {
+                return HttpNotFound();
+            }
+            return View(tag);
+        }
+
+        // GET: Tag/Create
+        public ActionResult Create()
+        {
+            return View();
+        }
+
+        // POST: Tag/Create
+        // To protect from overposting attacks, please enable the specific properties you want to bind to, for 
+        // more details see http://go.microsoft.com/fwlink/?LinkId=317598.
+        [HttpPost]
```

Comment:
Yo prefiero hacer el binding a un ViewModel que tenga solo los campos que quiero publicar. But I guess this works :)